config/initializers/imgproxy.rb

```diff
@@ -2,7 +2,12 @@
   # imgproxy endpoint
   #
   # Full URL to where your imgproxy lives.
-  config.endpoint = ApplicationConfig["IMGPROXY_ENDPOINT"]
+  #
+  config.endpoint = if Rails.env.production?
+                      URL.url("images")
```

Comment:
Why only for production? Why not use this for all environments?

Reply:
Having imgproxy set up with same domain as the rails app under `/images` require special openresty configuration. This gives more flexibility to local setups.

Reply:
Gotcha, it would be awesome if we could remove this somehow but since that might be hard we should add a comment about why it is here. It is always a little bit of a red flag to me when I see `if Rails.env.production?` statements in the code but at least a comment will explain a bit what is going on.